### PR TITLE
[86bx76wy1][utils] moved InvalidStatePattern to the flex-box package

### PR DESCRIPTION
**Pull request by ilyabrower**
## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [X] Nice improve.

## Checklist:

- [X] My code follows the code style of this project.
- [X] I have updated the documentation accordingly or it's not required.
- [X] Unit tests are not broken.
- [X] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: semcore/flex-box/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [5.19.0] - 2024-02-05
+
+### Added
+
+- `InvalidStatePattern` component.
```
Comment (msereniti) [inline semcore/flex-box/CHANGELOG.md:9]: Let's name it `InvalidStateBox` to align it with `Box` naming.

File: semcore/checkbox/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [7.21.1] - 2024-02-05
+
+### Changed
+
+- Use `InvalidStatePattern`.
```
Comment (msereniti) [inline semcore/checkbox/CHANGELOG.md:9]: Let's write it as "**Fixed** Invalid attribute errors in console that appears when component has invalid state"

File: semcore/utils/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.20.1] - 2024-02-05
+
+### Changed
+
+- Mark InvalidStatePattern as `deprecated` because of moving it to flex-box.
```
Comment (msereniti) [inline semcore/utils/CHANGELOG.md:9]: Let's write something like "Deprecated all `@semcore/ui/utils/lib/invalid-state-pattern/InvalidStatePattern`" exports.

File: semcore/flex-box/CHANGELOG.md (hunk 1 above)
Comment (ilyabrower) [reply semcore/flex-box/CHANGELOG.md:9]: fixed

File: semcore/checkbox/CHANGELOG.md (hunk 2 above)
Comment (ilyabrower) [reply semcore/checkbox/CHANGELOG.md:9]: fixe

File: semcore/utils/CHANGELOG.md (hunk 3 above)
Comment (ilyabrower) [reply semcore/utils/CHANGELOG.md:9]: fixed

**Review by msereniti — APPROVED** [review]

**Pull request merged**